moj-docker-deploy/files/ebs-tag.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python
+
+import boto
+import sys
+import boto.ec2
+import requests
+
```

Comment:
Boto is actually installed early on (https://github.com/ministryofjustice/bootstrap-salt/blob/master/bootstrap_salt/contrib/bootstrap.sh#L6)

No harm in being explicit though!